components/user-store/org.wso2.carbon.identity.user.store.configuration.ui/src/main/resources/web/userstore_config/remove-userstore-ajaxprocessor.jsp

```diff
@@ -49,8 +52,21 @@
 						CarbonConstants.CONFIGURATION_CONTEXT);
 		userStoreConfigAdminServiceClient = new UserStoreConfigAdminServiceClient(
 				cookie, backendServerURL, configContext);
+		List<UserStoreDTO> userStoreDTOList = new ArrayList();
+		
+		for (String properties : checkedList) {
```

Comment:
What are the parameters in checkedList? Is this correct to use as 'properties'? I think properties' is just one parameter name.